frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -195,6 +195,12 @@ entitydetails-GenericOriginalString--plural = Plural
 entitydetails-GenericOriginalString--singular = Singular
 
 
+## Rich editor
+## Renders a Rich editor for Fluent string editing
+
+fluenteditor-RichTranslationForm--plural-example = (e.g. <stress>{ $example }<stress>)
```

Comment:
I'd argue that the plural category should be part of this message, too, to allow for ordering or other grammar needed in this phrase.